src/app/model/user/user.ts

```diff
@@ -0,0 +1,17 @@
+import { UserTeams } from "./user-teams";
+
+
+export class User {
+
+    constructor(
+        public id: number,
+        public firstname: string,
+        public lastname: string,
+        public email: string,
+        public password: string,
+        public team: UserTeams[],
+        public profilImage: string,
+        public level: number,
+        public reward: number,
```

Comment:
if it refers to reward number please change property for rewardNumber, or we could be confuse thinking that a Reward object could be binded to this property